docusaurus/docs/protocol/upgrades/upgrade_list.md

```diff
@@ -18,21 +20,26 @@ The tables below provide a list of past and upcoming protocol upgrades. For more
-| [`v0.0.5`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.5) |    ❓    |    ❓     | ✅ (Alpha TestNet Participants Only) | ❓              |
-| [`v0.0.4`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.4) |    ❓    |    ❓     | ✅ (Alpha TestNet Participants Only) | ❓              |
+| Version                                                                      | Planned | Breaking |                                                          Requires Manual Intervention                                                          | Upgrade Height                                                                                                               |
+| ---------------------------------------------------------------------------- | :-----: | :------: | :--------------------------------------------------------------------------------------------------------------------------------------------: | ---------------------------------------------------------------------------------------------------------------------------- |
+| [`v0.0.10`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.10)   |    ✅    |    ✅     |                                                             ❌ (automatic upgrade)                                                              | ❓ [56860](https://shannon.testnet.pokt.network/poktroll/tx/4E201E5C397AB881F417266154C907D38404BE00BE9A443DE28E44A2B09C5CFB) |
+| [`v0.0.9-4`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9-4) |    ❌    |    ✅     |                    ⚠️ [follow manual upgrade instructions](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9-4) ⚠️                    | `46329`                                                                                                                      |
+| [`v0.0.9-3`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9-3) |    ❌    |    ✅     | ❌ Active Alpha TestNet Participants Only: [follow manual upgrade instructions](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9-3) | `17102`                                                                                                                      |
+| [`v0.0.9`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9)     |   N/A   |   N/A    |                                                               ❌ genesis version                                                                | N/A                                                                                                                          |
+### Syncing from genesis - manual steps
+<!-- TODO(@okdas): when the next cosmovisor version released with `https://github.com/cosmos/cosmos-sdk/pull/21790` included - provide automated solution (csv file + pre-downloaded binaries) that will add hot-fixes automatically, allowing to sync from block #1 without any intervention -->
```

Comment:
FYI that I believe this is live now: https://github.com/cosmos/cosmos-sdk/releases/tag/cosmovisor%2Fv1.7.0

Reply:
this is wonderful